domain/src/main/java/org/fao/geonet/repository/UserGroupRepositoryCustomImpl.java

```diff
@@ -78,8 +78,11 @@ public int deleteAllByIdAttribute(SingularAttribute<UserGroupId, Integer> idAttr
             }
             idString.append(id);
         }
-        final String qlString = "DELETE FROM " + UserGroup.class.getSimpleName() + " WHERE " + userIdPath + " IN (" + idString + ")";
-        final int deleted = _entityManager.createQuery(qlString).executeUpdate();
+        final String qlString = "DELETE FROM UserGroup WHERE :userIdPath IN ( :idString )";
+        final int deleted = _entityManager.createQuery(qlString)
+            .setParameter("userIdPath", userIdPath)
+            .setParameter("idString", idString)
+            .executeUpdate();
```

Comment:
This method isn't passing the test. Can you change it to:
```java
    public int deleteAllByIdAttribute(SingularAttribute<UserGroupId, Integer> idAttribute, Collection<Integer> ids) {
        String userIdPath = SortUtils.createPath(UserGroup_.id, idAttribute);

        final String qlString = String.format("DELETE FROM UserGroup WHERE %s IN :idString", userIdPath);
        final int deleted = _entityManager.createQuery(qlString)
            .setParameter("idString", ids)
            .executeUpdate();
        _entityManager.flush();
        _entityManager.clear();

        return deleted;
    }
```

Reply:
Should be now incorporated